oauth2_http/java/com/google/auth/oauth2/ServiceAccountJwtAccessCredentials.java

```diff
@@ -228,6 +241,28 @@ public static ServiceAccountJwtAccessCredentials fromStream(InputStream credenti
             + " Expecting '%s'.", fileType, SERVICE_ACCOUNT_FILE_TYPE));
   }
```

Comment:
fromJson doesn't appear to create / set tokenCache.

Reply:
it does via:
fromJson(Map, URI) -> fromPkcs8(...) -> new ServiceAccountJwtAccessCredentials